packages/code-studio/package-lock.json

```diff
@@ -1282,124 +1264,6 @@
 			"resolved": "https://registry.npmjs.org/@csstools/normalize.css/-/normalize.css-10.1.0.tgz",
 			"integrity": "sha512-ij4wRiunFfaJxjB0BdrYHIH8FxBJpOwNPhhAcunlmPdXudL1WQV1qoP9un6JsEBAgQH+7UXyyjh0g7jTxXK6tg=="
 		},
-		"@deephaven/chart": {
-			"version": "0.1.1",
-			"resolved": "https://registry.npmjs.org/@deephaven/chart/-/chart-0.1.1.tgz",
-			"integrity": "sha512-QH7Zgp4K5+NeRZCTdjTXK3gen91NfUB/Fw0oCuvcCuvWciegJXHW/RIeW9isCPUt6F7iB7K/fMTLv3Zxyz0odA==",
-			"requires": {
-				"plotly.js": "^1.54.7",
-				"react-plotly.js": "^2.4.0"
-			}
-		},
-		"@deephaven/components": {
```

Comment:
Weird that all our packages were removed from this package-lock

Reply:
I believe it is correct - since it's pulling from within the same repo